hazelcast/src/main/resources/hazelcast-config-4.1.xsd

```diff
@@ -3149,6 +3150,49 @@
         </xs:restriction>
     </xs:simpleType>
 
+    <xs:complexType name="persistent-memory">
+        <xs:annotation>
+            <xs:documentation>
+                Configuration for persistent memory (e.g. Intel Optane) devices.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:all>
+            <xs:element name="directories">
+                <xs:annotation>
+                    <xs:documentation>
+                        List of directories where the persistent memory
+                        is mounted to.
+                    </xs:documentation>
+                </xs:annotation>
+                <xs:complexType>
+                    <xs:choice maxOccurs="unbounded">
+                        <xs:element name="directory" type="persistent-memory-directory"/>
+                    </xs:choice>
+                </xs:complexType>
+            </xs:element>
+        </xs:all>
+    </xs:complexType>
+
+    <xs:complexType name="persistent-memory-directory">
+        <xs:annotation>
+            <xs:documentation>
+                The directory where persistent memory is mounted to.
```

Comment:
Can you copy-paste other documentation like it needing to be unique, other restrictions and -1 numa-node disabling numa-awareness? Same in client XSD.

Reply:
:+1: Added to member, client and spring configs.